### PR TITLE
installation updated

### PR DESCRIPTION
**Pull request by YogiSoni-lazy**

### CODE REVIEW

File: _book/.github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+name: Publish to GitHub Pages
```
Comment (rsriniva) [inline _book/.github/workflows/main.yml:1]: Where is this _book folder coming from and what is it used for? I can't recall antora needing this, or this folder being a part of the default antora site template. Please delete this folder

File: modules/install/pages/install.adoc [hunk 2]
```diff
@@ -4,28 +4,45 @@ Brief blurb about how JBoss EAP offers multiple ways to install...
 
 == Pre-requisites
 
-Talk about what pre-requisites are needed:
+* You have created an account on the Red Hat Customer Portal to download JBoss EAP files.
+* You have installed a https://access.redhat.com/articles/6961381[Red Hat supported OpenJDK 17]
```
Comment (rsriniva) [inline modules/install/pages/install.adoc:8]: Is it confirmed that openjdk 17 is the best option for EAP 8? No openjdk 21?

File: modules/install/pages/install.adoc [hunk 3]
```diff
@@ -4,28 +4,45 @@ Brief blurb about how JBoss EAP offers multiple ways to install...
 
 == Pre-requisites
 
-Talk about what pre-requisites are needed:
+* You have created an account on the Red Hat Customer Portal to download JBoss EAP files.
```
Comment (rsriniva) [inline modules/install/pages/install.adoc:7]: Add a link to the developer portal - the sign up/login page

File: modules/install/pages/install.adoc [hunk 4]
```diff
@@ -4,28 +4,45 @@ Brief blurb about how JBoss EAP offers multiple ways to install...
...
+* You have installed a https://access.redhat.com/articles/6961381[Red Hat supported OpenJDK 17]
+* Recommended to create a separate *jboss* user account with no shell access (/sbin/nologin)
+* Recommended to download all installation file under /opt partition. Ensure there is enough space under /opt partition (minimum 800MB).
+You have reviewed JBoss EAP 8.0 supported configurations.
```
Comment (rsriniva) [inline modules/install/pages/install.adoc:11]: Add link to the EAP 8 supported config

File: modules/install/pages/install.adoc [hunk 5]
```diff
@@ -4,28 +4,45 @@ Brief blurb about how JBoss EAP offers multiple ways to install...
...
+* You have access to the internet.
 
-* Red Hat supported OpenJDK 17 or higher (21 recommended?)
-* Recommended to create a separate *jboss* user account with no shell access (/sbin/nologin) 
-* Recommended to install under /opt. Can be a dedicated partition. Ensure there is enough space (suggest a good minimum number)
+== Red Hat JBoss Enterprise Application Platform 8 Installation Methods
+
+You can install JBoss EAP 8.0 by using one of the following installation methods:
```
Comment (rsriniva) [inline modules/install/pages/install.adoc:16]: Can you add a few words either here or in the individual sections about why you would use one over the other - for ex the JAR installer needs a UI, while the zip install does not etc

File: modules/install/pages/install.adoc [hunk 6]
```diff
@@ -4,28 +4,45 @@ Brief blurb about how JBoss EAP offers multiple ways to install...
...
+
+* *JBoss EAP using the Zip file*
+* *JBoss EAP using the JAR installer*
+* *JBoss EAP using archive installation method*
```
Comment (rsriniva) [inline modules/install/pages/install.adoc:20]: what is "archive installation"?

File: modules/install/pages/install.adoc [hunk 7]
```diff
@@ -4,28 +4,45 @@ Brief blurb about how JBoss EAP offers multiple ways to install...
...
+* *JBoss EAP by using the RPM installation method*
 
 == Installing JBoss EAP using the Zip file
 
-Brief overview only. Details in lab section below....
+* This method can also be called jboss-eap-installation-manager installation method.
```
Comment (rsriniva) [inline modules/install/pages/install.adoc:25]: Are you sure about this? is "jboss-eap-installation-manager" an official term used in the docs?

File: modules/install/pages/install.adoc [hunk 8]
```diff
@@ -4,28 +4,45 @@ Brief blurb about how JBoss EAP offers multiple ways to install...
...
+* The zip file contains all the libraries, configuration files, and scripts needed to start JBoss EAP, without further customization.
 
 == Installing JBoss EAP using the JAR installer
 
-Brief overview only. Details in lab section below....
+* This method can also be called the GUI installer installation method.
+* In this method, the jar file is a Java based application that is responsible for providing a step-by-step way to customize the environment setup.
+* Due to its portable nature, you can use it on multiple platforms and it requires OpenJDK 17 installed on the host OS.
+* It can be run using either a graphical interface or a text based console.
+* Due to its flexible nature, the installer can install JBoss EAP by using a response file, with all the options passed as an XML file.
```
Comment (rsriniva) [inline modules/install/pages/install.adoc:35]: Add a NOTE that this advanced method with response files will be covered in the Advanced EAP Admin/2nd course in the path. Also add a sentence about ansible middleware and that we will cover in course 2

File: modules/install/pages/install.adoc [hunk 9]
```diff
@@ -44,23 +61,165 @@ Brief blurb about uninstall.....
 
 == References
 
-* Links to all relevant product docs
+* https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/8.0/html/red_hat_jboss_enterprise_application_platform_installation_methods/index[Red Hat JBoss Enterprise Application Platform Installation Methods]
+
 
 == Lab 1: Installing JBoss EAP using the Zip file
```
Comment (rsriniva) [inline modules/install/pages/install.adoc:67]: add a NOTE saying that the install instructions are for RHEL 9 and that students should adjust the instructions for their operating system

File: modules/install/pages/install.adoc [hunk 10]
```diff
@@ -44,23 +61,165 @@ Brief blurb about uninstall.....
...
+[subs="+quotes,+macros"]
+----
+[root@localhost opt]# cd jboss-eap-installation-manager-1.1.6.Final-redhat-00001/bin
+----
+
+* Install JBoss EAP by running the following command:
++
+[subs="+quotes,+macros"]
+----
+[root@localhost bin]# ./jboss-eap-installation-manager.sh install --profile eap-8.0 --dir eap-8
```
Comment (rsriniva) [inline modules/install/pages/install.adoc:103]: you have not explained what --profile eap-8.0 means? what other options are available? link to product docs where flags for this script are provided? why is it useful?

File: modules/install/pages/install.adoc [hunk 11]
```diff
@@ -44,23 +61,165 @@ Brief blurb about uninstall.....
...
+----
+
+* Start the JBoss EAP using the start script present under `/opt/jboss-eap-installation-manager-1.1.6.Final-redhat-00001/bin/eap-8/bin/standalone.sh`
++
+[subs="+quotes,+macros"]
+----
+[root@localhost ~]# /opt/jboss-eap-installation-manager-1.1.6.Final-redhat-00001/bin/eap-8/bin/standalone.sh
+----
+
+* Navigate to the JBOSS console at http://localhost:8080
```
Comment (rsriniva) [inline modules/install/pages/install.adoc:130]: We discussed this - for RHEL 9 VMs replace localhost with the NAT IP address of Linux VM.

File: modules/install/pages/install.adoc [hunk 12]
```diff
@@ -44,23 +61,165 @@ Brief blurb about uninstall.....
...
-.. Step 2.b
-. Step 3
+* Install OpenJDK 17 if not already installed:
++
+[subs="+quotes,+macros"]
+----
+[root@localhost ~]# sudo yum install java-17-openjdk -y
+----
+
+* Install the GUI if not already installed on RHEL 9:
```
Comment (rsriniva) [inline modules/install/pages/install.adoc:146]: duplicate step? WDYM by "GUI"? remove?

File: modules/install/pages/install.adoc [hunk 13]
```diff
@@ -44,23 +61,165 @@ Brief blurb about uninstall.....
...
+[root@localhost ~]# cd Downloads/
+[root@localhost Downloads]# ls -l jboss-eap-8.0.0-installer.jar
+-rw-r--r--. 1 root root 28141889 Apr  1 09:35 jboss-eap-8.0.0-installer.jar
+----
+
+* Run the graphical installer using the following command:
++
+[subs="+quotes,+macros"]
+----
+[root@localhost Downloads]# java -jar jboss-eap-8.0.0-installer.jar
```
Comment (rsriniva) [inline modules/install/pages/install.adoc:175]: I think you need to launch the installer as the jboss user? check AD248. if installing as root, then you need to chmod the folders where you installed EAP under /opt

File: modules/install/pages/install.adoc [hunk 14]
```diff
@@ -44,23 +61,165 @@ Brief blurb about uninstall.....
...
+----
+
+* Choose your preferred language for the installer, and then click OK.
+
+*  Agree with the prompt for The EULA for RED HAT JBOSS MIDDLEWARE by selecting "I accept the terms of this license agreement", and then click Next.
+
+* Select the installation path or choose a Maven source repository for JBoss EAP, and then click Next.
++
+image::assets/images/loc.png[align="center"]
```
Comment (rsriniva) [inline modules/install/pages/install.adoc:184]: none of the images are rendering. antora expects images under the module name. so create an "images" folder under modules/install and copy all the images into it. the correct asciidoc way to refer to images is

```image::rhods_install1.png[title=Search for OpenShift AI operator,width=800]```

File: modules/install/pages/install.adoc [hunk 15]
```diff
@@ -44,23 +61,165 @@ Brief blurb about uninstall.....
...
+
+* Once the progressing completes, select `Next`:
++
+image::assets/images/complete2.png[align="center"]
+
+* Finally, select `Done`:
++
+image::assets/images/complete3.png[align="center"]
+
+* Start the JBoss EAP using the start script present under `/home/jboss/EAP-8.0.0/bin/standalone.sh`
```
Comment (rsriniva) [inline modules/install/pages/install.adoc:214]: i think you should install EAP under /opt as the jboss user and the resulting folders should be owned by jboss:jboss.

File: modules/install/pages/install.adoc [hunk 16]
```diff
@@ -44,23 +61,165 @@ Brief blurb about uninstall.....
...
++
+[subs="+quotes,+macros"]
+----
+[root@localhost ~]# /home/jboss/EAP-8.0.0/bin/standalone.sh
```
Comment (rsriniva) [inline modules/install/pages/install.adoc:218]: show some output that EAP started successfully

File: modules/install/pages/install.adoc [hunk 17]
```diff
@@ -44,23 +61,165 @@ Brief blurb about uninstall.....
...
+----
+
+* Navigate to the JBOSS console at http://localhost:8080
```
Comment (rsriniva) [inline modules/install/pages/install.adoc:221]: see previous comment about localhost vs NAT IP

File: modules/install/pages/install.adoc (hunk 14 above)
Comment (rsriniva) [reply modules/install/pages/install.adoc:184]: also do not forget to check local rendering using the npm commands. Antora works differently than gitbook. Delete all gitbook related config and files

**Review by rsriniva — CHANGES_REQUESTED** [review]
An excellent start! You are on the right track. Please address PR comments and fill out the rest of the section

File: modules/install/pages/install.adoc (hunk 5 above)
Comment (rsriniva) [reply modules/install/pages/install.adoc:16]: Please also add the fact that you can run EAP container images using podman on RHEL 9. Link to the "official" EAP 8 base images built on UBI by engineering. Add that the base image can be customized by customers to adapt to their environment

File: modules/install/pages/install.adoc (hunk 2 above)
Comment (YogiSoni-lazy) [reply modules/install/pages/install.adoc:8]: Yes openjdk 17 is supported and not above. Here's the KCS link : https://access.redhat.com/solutions/6137992

File: modules/install/pages/install.adoc (hunk 3 above)
Comment (YogiSoni-lazy) [reply modules/install/pages/install.adoc:7]: add the link

File: modules/install/pages/install.adoc (hunk 4 above)
Comment (YogiSoni-lazy) [reply modules/install/pages/install.adoc:11]: Add the link: https://access.redhat.com/articles/6961381

File: modules/install/pages/install.adoc (hunk 14 above)
Comment (YogiSoni-lazy) [reply modules/install/pages/install.adoc:184]: Deleted gitbook related configs

Comment (YogiSoni-lazy) [reply modules/install/pages/install.adoc:184]: I have created "images" folder under modules/install

I see it is searching as: 
[2024-04-03T09:30:50.672Z]  "GET /eap-admin1/8/install/images/complete2.png" Error (404): "Not found"
[2024-04-03T09:30:50.672Z]  "GET /eap-admin1/8/install/images/complete3.png" Error (404): "Not found"

Not sure why the error is still there, today when we sync, we can discuss

**Comment by rsriniva** [comment]
@YogiSoni-lazy - as discussed, please delete this PR and branch

**Pull request closed without merging**